JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,15 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  let hours, minutes, second;
+  hours = Math.trunc(seconds / 3600);
+  leftSecond = seconds % 3600;
+  minutes = Math.trunc(leftSecond / 60);
+  second = leftSecond % 60;
+  //This formats the respond, if the number is less than 9, add a 0 in front of it.
+  return `${hours <= 9 ? '0' + hours : hours}:${
+    minutes <= 9 ? '0' + minutes : minutes
+  }:${second <= 9 ? '0' + second : second}`;
```

Comment:
we could use nested templete strings. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Maybe we could throw an error when a negative number is given.

Reply:
Spaces help us to improve readibility, please try to add blank spaces. In this scenario  we could have three blocks declaration, definition, return.